diffpy/pdfmorph/pdfmorphapp.py

```diff
@@ -254,7 +254,8 @@ def main():
 
     if opts.plot:
         pairlist = [chain.xyobjout, chain.xyrefout]
-        labels = ["objective", "reference"]
+        labels = [pargs[0], pargs[1]]
+        print(pargs[0], pargs[1])
```

Comment:
please remove.  I guess this was a debugging line

Reply:
Which template are you referring to here? I have the template files from the paper but is there something else? Also, where is the style sheet located? Is it the same as the group meeting one? Also, yes, that was a debugging line will remove it before doing the next PR.

Reply:
This is standard rever release workflow.  Please refer to the regro/rever documentation and/or talk to songsheng.  You can see examples in  the regolith repo and there may be something in the wiki on this, I am not sure.  I presume you looked there already.

I don't see a comment on the style-sheet.  Can you figure that one out?

Reply:
Alright will do. For the style-sheet I just wanted to check where it was located at, I can figure out the rest myself. Thanks

Reply:
sound good. Please check the wiki and get back if you can't find info there on the style-sheet.